### DISCUSSION TITLE
Add custom meta on specific openapi doc

### DISCUSSION THREAD

**Discussion by SoukblacK** [rohit-gohri/redocusaurus#363] [General]
Hi !

I have 2 configuration of openapi and i want to add meta robots noindex only for one.
I dont find how to do that.

Someone have an idea ?

Thanks

```
specs: [
	{
		spec: "openapi/xxxx.yaml",
		route: "/reference/xxxx",
		layout: {
			title: "xxxx",
			noFooter: true,
		},
	},
	{
		spec: "openapi/zzzz.yaml",
		route: "/reference/zzzz",
		layout: {
			title: "zzzz",
			noFooter: true,
		},
	},
],
```

**Comment by rohit-gohri** [comment]
That could be good feature to support, but currently not possible without using custom React page (like this https://github.com/rohit-gohri/redocusaurus/blob/main/website/src/pages/examples/custom-page/index.jsx) and adding your own <Head> tags

**Reply by malathikrishnap** [reply]
I have a similar situation to add a custom meta tag to the json file to enable SEO.
But, Redocly is not supporting adding meta tags.

I tried your suggested way of creating a React component, even there I'm unable to add a random tag, for example "guidename".
It's accepting only pre-defined tags, such as, "description".

Is there a way to add a custom meta tag to the HTML output of the json file?
